### PR TITLE
LPS-139484 Add comma as a decimal separator in rules conditions in decimal numeric field

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://issues.liferay.com/browse/LPS-139484

### CODE REVIEW

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js [hunk 1]
```diff
@@ -31,6 +31,34 @@ const localDataStorage = {
...
+			const fieldInfo = condition.operands[0];
+			const fieldValue = condition.operands[1];
+
+			if (
+				fieldInfo.value.includes('Numeric') &&
+				typeof fieldValue.value == 'string'
+			) {
+				if (
+					fieldValue.value.includes(',') ||
+					fieldValue.value.includes('.')
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js:47]: @danvb112, maybe, we should move this specific check to a dedicated method. Why?

Consider the scenario in which a new decimal separator will be supported then having this logic inside a method will be easier to evolve and maintain this code, what do you think?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js [hunk 2]
```diff
@@ -62,6 +90,7 @@ export const RuleEditor = ({onCancel, onSave, rule, ...otherProps}) => {
 						disabled={disabled}
 						displayType="primary"
 						onClick={() => {
+							convertNumericDataToNumber(localDataStorage.rule);
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js:93]: By the way, @danvb112, isn't the root cause of this issue the validation that prevents non digits characters, like `,` and `.`, to be added into a Numeric Field?

Perhaps, we should only adjust this validation to support the decimal separators too, what do you think?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js (hunk 1 above)
Comment (danvb112) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/RuleEditor.es.js:47]: @natocesarrego I totally agree with you, but this specific method should be created in this same file?

**Comment by liferay-forms** [comment]
Hey @danvb112, can you please check if these failures are related to your changes? Thanks

**Pull request closed without merging**